src/test/java/ay2021s1_cs2103_w16_3/finesse/logic/commands/ListExpenseCommandTest.java

```diff
@@ -30,30 +28,11 @@ public void setUp() {
     }
 
     @Test
-    public void execute_showsExpenseOnly() {
-        expectedModel.updateFilteredTransactionList(PREDICATE_SHOW_ALL_EXPENSES);
+    public void execute_showsAllExpensesOnly() {
+        expectedModel.updateFilteredExpenseList(PREDICATE_SHOW_ALL_TRANSACTIONS);
         assertCommandSuccess(new ListExpenseCommand(), model,
                 new CommandResult(ListExpenseCommand.MESSAGE_SUCCESS, Tab.EXPENSES), expectedModel);
-    }
-
-    @Test
-    public void execute_hasSomeExpense() {
-        model.addTransaction(new TransactionBuilder().buildExpense());
-        expectedModel = new ModelManager(model.getFinanceTracker(), new UserPrefs());
-        expectedModel.updateFilteredTransactionList(transaction -> transaction instanceof Expense);
-        assertCommandSuccess(new ListExpenseCommand(), model,
-                new CommandResult(ListExpenseCommand.MESSAGE_SUCCESS, Tab.EXPENSES), expectedModel);
-        assertEquals(model.getFilteredTransactionList().size(), 1);
-    }
-
-    @Test
-    public void execute_hasSomeNonExpense() {
-        model.addTransaction(new TransactionBuilder().buildIncome());
-        expectedModel = new ModelManager(model.getFinanceTracker(), new UserPrefs());
-        expectedModel.updateFilteredTransactionList(transaction -> transaction instanceof Expense);
-        assertCommandSuccess(new ListExpenseCommand(), model,
-                new CommandResult(ListExpenseCommand.MESSAGE_SUCCESS, Tab.EXPENSES), expectedModel);
-        assertEquals(model.getFilteredTransactionList().size(), 0);
+        assertEquals(model.getFilteredExpenseList().size(), 7);
```

Comment:
Is it possible to not hard-code the value `7`? Same as in `ListIncomeCommandTest` and `ListTransactionCommandTest`

Reply:
This value depends on the number of values in our test data though. How do you propose we get around this without hard-coding?

Reply:
Perhaps `getTypicalFinanceTracker().getExpenseList().size()`?

By the way, it doesn't really matter here but the expected value is supposed to be the first argument to `assertEquals`.

Reply:
```suggestion
        assertEquals(model.getFilteredExpenseList().size(), getTypicalExpenses().size());
```
This together with an additional import statement:
```
import static ay2021s1_cs2103_w16_3.finesse.testutil.TypicalTransactions.getTypicalExpenses;
```